### PR TITLE
add mainPage and loginTest

### PR DESCRIPTION
**Pull request by Raubezh**

### CODE REVIEW

File: src/main/java/MainPage.java [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+    public MainPage clickPasswordButton(){
+        driver.findElement(passwordButton).click();
+        return new MainPage(driver);
+    }
+
+    public MainPage login(String email, String password){
+        this.typeEmail(email);
+        this.clickPasswordButton();
+        this.typePassword(password);
+        return new MainPage(driver);
```
Comment (nchedanova) [inline src/main/java/MainPage.java]: pls add signInButton method

File: src/main/java/MainPage.java [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
-    public MainPage(WebDriver driver){
+    public MainPage(WebDriver driver) {
         this.driver = driver;
     }
+
+    private By mailField = By.xpath("//input[@id='mailbox:login']");
+    private By dropdownMailSelect = By.xpath("//div[@id='mailbox:select_container']");
+    private By passwordButton = By.xpath("//input[@value='Ввести пароль']");
+    private By passwordField = By.xpath("//input[@id='mailbox:password']");
+
```
Comment (nchedanova) [inline src/main/java/MainPage.java:16]: add SignInButton

**Review by nchedanova — CHANGES_REQUESTED** [review]
pls add what I'm writing in the comments

File: src/main/java/MainPage.java (hunk 1 above)
Comment (Raubezh) [reply src/main/java/MainPage.java]: done

**Review by nchedanova — APPROVED** [review]
ok

**Pull request merged**